test/parallel/test-net-dns-lookup.js

```diff
@@ -29,12 +29,12 @@ const server = net.createServer(function(client) {
   server.close();
 });
 
-server.listen(0, '127.0.0.1', common.mustCall(function() {
+server.listen(0, 'localhost', common.mustCall(function() {
```

Comment:
```suggestion
server.listen(0, common.localhostIP, common.mustCall(function() {
```

Reply:
In that case it would only listen on either of the localhost addresses, but not dualstack. Don't we want dualstack?

Reply:
Il you put `localhost` in the DNS test that precisely tests that `localhost` is resolved to the loopback address, what are you really testing? Or maybe I misunderstood what the test is trying to do, but I think this change is wrong.

Reply:
Good question. I don't know. I thought it is connecting to the inspector server.